src/components/Login/LoginModal.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react';
+import { LOGIN_CONFIG } from '../../config';
+import { Link } from 'react-router-dom';
+import './LoginModal.scss';
+
+function Login({ closeTargetModal }) {
+  const [inputs, setInputs] = useState({
+    userName: '',
+    password: '',
+  });
+
+  const handleChangeInputs = ({ target: { name, value } }) =>
+    setInputs(prev => ({ ...prev, [name]: value }));
+
+  const validateLogin = ({ userName, password }) => {
+    if (!(userName.length && password.length < 1)) {
+      throw new Error('아이디 혹은 비밀번호를 입력해주세요. ');
+    }
+  };
+
+  const handleClickLogin = value => {
+    try {
+      validateLogin(value);
+
+      fetch(`${LOGIN_CONFIG.api}`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json;charset=utf-8',
+        },
+        body: JSON.stringify(value),
+      })
+        .then(response => {
+          if (response.ok) {
+            return response.json();
+          }
+        })
+        .then(data => {
+          localStorage.setItem('token', data.accessToken);
+          localStorage.setItem('fullName', data.fullName);
+
+          alert('로그인 성공');
+          handleCloseModal();
+        })
+        .catch(e => console.error(e));
+    } catch (e) {
+      alert('아이디 혹은 비밀번호를 확인해주세요.');
+    }
+  };
+
+  const handleCloseModal = () => closeTargetModal('login');
+
+  return (
+    <div
+      className="loginWrapper"
+      onClick={() => {
+        handleCloseModal();
+      }}
```

Comment:
```suggestion
      onClick={handleCloseModal}
```
이렇게 구현하는 방향으로 하면 되겠습니다!

Reply:
수정 했습니다!